ct/pocketbase.sh

```diff
@@ -52,6 +52,36 @@ function default_settings() {
   echo_default
 }
 
+function update_script() {
+  header_info
+  check_container_storage
+  check_container_resources
+  if [[ ! -f /etc/systemd/system/pocketbase.service ]]; then msg_error "No ${APP} Installation Found!"; exit; fi
+
+  RELEASE=$(curl -s https://api.github.com/repos/pocketbase/pocketbase/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+
+  if [[ ! -f /opt/${APP}_version.txt || "${RELEASE}" != "$(cat /opt/${APP}_version.txt)" ]]; then
+    msg_info "Stopping ${APP}"
+    systemctl stop pocketbase.service
+    msg_ok "Stopped ${APP}"
+
+    msg_info "Updating $APP to ${RELEASE}"
+    wget -q https://github.com/pocketbase/pocketbase/releases/download/v${RELEASE}/pocketbase_${RELEASE}_linux_amd64.zip -O /tmp/pocketbase.zip
+    mkdir -p /opt/pocketbase/{pb_public,pb_migrations,pb_hooks}
+    unzip -q -o /tmp/pocketbase.zip -d /opt/pocketbase
```

Comment:
Make sure no migration steps are needed after replacing the executable

Reply:
So, I just learned that pocketbase had a `pocketbase update` command that backs up the data directory, updates the binary _and_ runs with `--automigrate` by default.

I will switch the update script to using that, in the hopes that this is what you meant.

Reply:
Done: [`a604a97` (#535)](https://github.com/community-scripts/ProxmoxVE/pull/535/commits/a604a974659f73e7ac61a9979567f97ae70c5d65)